src/virtac/create_csv.py

```diff
@@ -349,7 +363,7 @@ def write_data_to_file(data, filename, ring_mode):
     )
     with open(filepath, "w", newline="") as file:
         csv_writer = csv.writer(file)
-        csv_writer.writerows(data)
+        csv_writer.writerows([data[0]] + sorted(data[1:]))
```

Comment:
I assume this is done to order the elements while preserving the column titles? You could either pass in the titles as an extra argument, or be more explicit in the code, e.g.

```suggestion
        column_titles = data[0]
        sorted_output = [column_titles] + sorted(data[1:])
        csv_writer.writerows(sorted_output)
```

Reply:
Yes, I'll do something to make it clearer